cabal2nix/src/Distribution/Nixpkgs/Haskell/FromCabal/PostProcess.hs

```diff
@@ -123,7 +123,7 @@ hooks =
   , ("hlibgit2 >= 0.18.0.14", set (testDepends . tool . contains (pkg "git")) True)
   , ("hmatrix < 0.18.1.1", set phaseOverrides "preConfigure = \"sed -i hmatrix.cabal -e '/\\\\/usr\\\\//D'\";")
   , ("holy-project", set doCheck False)         -- attempts to access the network
-  , ("hoogle", set testTarget "--test-option=--no-net")
+  , ("hoogle", set testTargets ["--test-option=--no-net"])
```

Comment:
This should be `testFlags` instead, but I suspect cabal2nix doesn't know about it yet. Maybe something to take care of in a follow up change / #504.